### ISSUE TITLE
Auto logout

### ISSUE THREAD

**Issue by sethlyons** [bitwarden/browser#91]
In addition to auto-lock, an auto-logoff would be great.  Example:  I have auto-lock set for fifteen minutes of inactivity.  If, however, I'm away from my computer for a few hours, I would like to be forced to reauthenticate (since that's where my 2FA is enforced), instead of simply just entering my master password.  Something like auto-lock at fifteen minutes and auto-logoff after an hour.
Thanks!

**Comment by jungle-boogie** [comment]
I support this! I'd say this is almost a duplicate of https://github.com/bitwarden/browser/issues/173

**Comment by baenglund** [comment]
@jungle-boogie I agree. I think these two are should me implemented together. The other thread has a lot of good ideas on the implementation of it such as [this comment](https://github.com/bitwarden/browser/issues/173#issuecomment-354970762).

That combined with the idea of @sethlyons would be a good solution.
> Something like auto-lock at fifteen minutes and auto-logoff after an hour.

Between auto "lock" and auto "logout" you have the convenience of just the password and security of 2FA when you leave it for an extended period of time.
